Content/Serializer/MediaSerializer.php

```diff
@@ -94,15 +94,19 @@ public function serialize(MediaInterface $media, string $locale, ?SerializationC
         $preferredExtension = $this->imageConverter->getSupportedOutputImageFormats($formatMediaApi->getMimeType())[0] ?? null;
         if ($preferredExtension) {
```

Comment:
I think `formatUri`, `formatPreferredExtension` should be `null` if there is no preferredExtension as I think that indicates that there are no formats available for this media. E.g. `.doc`, `.xls`, ...